### ISSUE TITLE
Regression plot changes during the training

### ISSUE THREAD

**Issue by utausheva** [stared/livelossplot#5]
Live loss plot work almost perfect. But in regression problems there is the interest in Live Regression Plot, that can show us how predicted curve tends to real test curve during the training epochs. 

Is it hard for you to do this kind callback based on livelossplot callback class?

**Comment by utausheva** [comment]
![69cfdeb9998a4851baa06d6fb6eb0029 4](https://user-images.githubusercontent.com/22944267/37663031-62f784f6-2c69-11e8-83e1-8e652620193d.jpg)
![69cfdeb9998a4851baa06d6fb6eb0029 3](https://user-images.githubusercontent.com/22944267/37663034-652fe984-2c69-11e8-8e35-0fc2d46b3ea3.jpg)
![69cfdeb9998a4851baa06d6fb6eb0029 2](https://user-images.githubusercontent.com/22944267/37663037-66a663e2-2c69-11e8-95ad-ff43272c9c75.jpg)
![69cfdeb9998a4851baa06d6fb6eb0029](https://user-images.githubusercontent.com/22944267/37663040-6847f95e-2c69-11e8-8b50-86ea0d4e1a88.jpg)

**Comment by stared** [comment]
@utausheva, could you provide full code? 

Though, if it is not a loss function, it seems to be outside of scope of this project. At the same time, feel free to reuse any part of https://github.com/stared/livelossplot/blob/master/livelossplot/core.py (look at clear_output + standard matplotlib functions).

**Comment by utausheva** [comment]
it's a pity(( I'll try to do this on my own

**Comment by stared** [comment]
See #60 for an example.
